db.php

```diff
@@ -0,0 +1,16 @@
+<?php
+$db = [
+    'host' => 'localhost',
+    'user' => 'root',
+    'password' => 'root',
+    'name' => 'doingsdone'
+];
+
+
+$connect = mysqli_connect($db['host'], $db['user'], $db['password'], $db['name']);
+
+if (!$connect) {
+    die('Ошибка подключения: ' . mysqli_connect_error() . mysqli_errno());
+} else {
+    mysqli_set_charset($connect, "utf8");
+}
```

Comment:
```suggestion
}

mysqli_set_charset($connect, "utf8");

```
В подобных случаях else можно не писать, т.к. в теле if будет die(), т.е. программа закончит свою работу. Соответственно, всё что за телом if будет только в случае else.